onionshare/common.py

```diff
@@ -58,6 +59,11 @@ def get_platform():
     """
     return platform.system()
 
+def platform_is_unixy():
+    """
+    Returns true if the platform we're running on is Linux or BSD
+    """
+    return True if re.match('^(Linux|.*BSD)$', get_platform()) else False
```

Comment:
Could simplify this a bit:

```python
def platform_is_unixy():
    return re.search('(Linux|BSD)', get_platform())
```

Or you could test whether you're on a POSIX system (the `posix` import is only available on such a system):

```python
def platform_is_unixy():
    try:
        import posix
        return True
    except ImportError:
        return False
```